src/Microsoft.DocAsCode.Dfm.VscPreview/DfmServiceProvider.cs

```diff
@@ -0,0 +1,58 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace DfmProcess
+{
+    using System.Collections.Generic;
+    using System.Collections.Immutable;
+    using System.Composition;
+
+    using Microsoft.DocAsCode.Dfm;
+    using Microsoft.DocAsCode.MarkdownLite;
+    using Microsoft.DocAsCode.Plugins;
+
+
+    [Export("dfm", typeof(IMarkdownServiceProvider))]
```

Comment:
it will be used in the JsonMarkup.But I can remove it now.

---
In reply to: [73892934](https://github.com/dotnet/docfx/pull/583#discussion_r73892934) [](ancestors = 73892934)